test/tests/unit/backend.ts

```diff
@@ -1,8 +1,12 @@
 import chai from "chai";
 import { redisClient } from "../../../backend/redis";
+import { updateCache, catchup, LedgerRecord } from "../../../backend/ledgers";
 const v1 = require("../../../backend/lumens");
 const v2v3 = require("../../../backend/v2v3/lumens");
 
+const REDIS_LEDGER_KEY = "ledgers_test";
+const REDIS_PAGING_TOKEN_KEY = "paging_token_test";
```

Comment:
These were defined earlier in the code, maybe we could move them to a separate file (`constants.ts`, for example)? That way we can import them where needed.

Reply:
🤔  I'd like to keep these in the test file for now since they're constants specific to these tests. Maybe if they're used more widely for tests we could add to a different file. But the naming could probably be better, changed to make it clear used for the redis testing only: https://github.com/stellar/dashboard/pull/209/commits/85e319459c7a103a361983cd1c758de1540c9524